src/app/users/users.component.html

```diff
@@ -31,13 +66,13 @@ <h1 i18n>Users</h1>
         <div *ngIf="!element.admin">
           <mat-icon  *ngIf="element.roles.length === 0" (click)="addRole(element)">lock</mat-icon>
           <mat-icon *ngIf="element.roles.length > 0" (click)="removeRole(element)">lock_open</mat-icon>
-          <a routerLink="/users/profile/{{element.name}}" i18n mat-raised-button color="primary">View Profile</a>
+          <a routerLink="/users/profile/{{element.name}}" mat-button i18n>View Profile

```

Comment:
Lets make this a `mat-raised-button`.  It'll make it clearer that the folder icon & the text are the same.